workspace/solution.go

```diff
@@ -19,6 +19,7 @@ type Solution struct {
 	Track       string     `json:"track"`
 	Exercise    string     `json:"exercise"`
 	ID          string     `json:"id"`
+	Team        string     `json:"team,omitempty"`
```

Comment:
what is the intended empty state for Team “”, null? I ask because I am wondering if this should be of type *string. 

If the empty state is “” then your example of “null” in the test should be changed.

Reply:
The empty state of the solution is `""`. The empty state of the payload we receive from the server is `null`.